### DISCUSSION TITLE
Add option to disable the shortcut icon ✨

### DISCUSSION THREAD

**Discussion by larsch** [community/community#124633] [Copilot Conversations]
### Select Topic Area

Product Feedback

### Body

Please add option to disable the inline Copilot shortcut ✨.

The shortcut icon to "Run: Generate using Copilot" shows up, even when Copilot is disabled (e.g. in markdown/plaintext files).

This is very noisy and sometimes makes text unreadable when it appears on top of text. 

Consider disabling it by default for language files where Copilot is already disabled anyway (markdown, plaintext, scminput).

IDE: vscode

**Comment by digitarald** [comment]
https://github.com/microsoft/vscode-copilot-release/issues/865 discusses options, where `"editor.lightbulb.enabled": "onCode"` might be what you want.

**Reply by justschen** [reply]
In addition to `onCode`, the widget will now use a "best case" placement and can spawn in the gutter as to not block any code! This is released in our latest Insiders already and is shipping in 1.92

**Comment by Mubashir0011a** [comment]
Select Topic Area:
Product Feedback

Body:
Thank you for the feedback regarding the inline Copilot shortcut. We understand how this feature can be disruptive, especially when Copilot is disabled. Your suggestion to disable it by default for certain file types, such as markdown and plaintext, is valuable and could improve readability.

In the meantime, if you're looking for inspiration or want to explore how different features and settings can enhance your coding experience, you might find the underwater temples of Bali intriguing. Just as these temples offer a unique and peaceful escape, optimizing your development environment can greatly enhance your productivity. For a refreshing break, consider checking out [Bali's underwater temples](https://fitcoding.com/bali-underwater-temples/).

We appreciate your input and will take it into consideration for future updates.
